ush/generate_FV3LAM_wflow.py

```diff
@@ -294,15 +282,15 @@ def generate_FV3LAM_wflow(ushdir, logfile: str = "log.generate_FV3LAM_wflow", de
     )
     #
     # Set npx and npy, which are just NX plus 1 and NY plus 1, respectively.
-    # These need to be set in the FV3-LAM Fortran namelist file.  They represent
+    # These need to be set in the FV3-LAM Fortran namelist file.  They represen
```

Comment:
The next three changes in this script all cut the last character from the last word in the line.